src/pages/ContentfulPage.vue

```diff
@@ -215,6 +237,34 @@ export default {
 			};
 		},
 		preFetch(config, client, args) {
+			if (args?.route?.path === '/lp/home-ml') {
+				client.query({ query: pageQuery }).then(() => {
```

Comment:
This promise needs to be returned, otherwise it won't prevent the contentful entries fro being fetched down on line 268. It might be cleaner to write this using async/await:
```
preFetch: async function(config, client, args) {
	if (args?.route?.path === '/lp/home-ml') {
		const result = await expQuery...
		const version = result?.data?.experiment?.version;
		const { enabled } = getExperimentSettingCached(client, manualLendingLPExpKey);
		
		if (enabled) {
			...
		}
	} else {
		return client.query({
				query: contentfulEntries,
				...rest of existing prefetch

Reply:
Thanks @emuvente ! Just one question regarding `pageQuery` that should be executed before the experiment query, right? As:

```
...
await client.query({ query: pageQuery });
const result = await client.query({ query: experimentQuery, variables: { id: manualLendingLPExpKey } })
...
```